doc/api/tls.md

```diff
@@ -628,6 +641,19 @@ For example:
 
 If the peer does not provide a certificate, an empty object will be returned.
 
+### tlsSocket.getPeerFinished()
+
+* Returns: {Buffer|undefined} The latest `Finished` message that is expected
+or has actually been received from the socket as part of a SSL/TLS handshake,
+or `undefined` if there is no `Finished` message so far.
+
+As the `Finished` messages are message digests of the complete handshake
+(with a total of 192 bits for TLS 1.0 and more for SSL 3.0), they can
+be used for external authentication procedures when the authentication
+provided by SSL/TLS is not desired or is not enough.
+
+Corresponds to the `SSL_get_peer_finished` routine in OpenSSL.
```

Comment:
I think the docs could be more explicit that it's for implementing tls-unique channel binding from RFC 5929.

(That _is_ the intended use case, right?)

Reply:
@bnoordhuis While [RFC 5929][1] might be a use case for the introduced `tlsSocket.getFinished()` and `tlsSocket.getPeerFinished()` methods, that is not exactly what `rippled` does. Should I still reference RFC 5929 in documentation?

[1]: https://tools.ietf.org/html/rfc5929

Reply:
I would lean towards 'yes'.  Outside of rippled that seems like the primary (and possibly only) use case.

Reply:
@bnoordhuis Referenced RFC 5929 in the amended commit.

Reply:
Definitely agree that the docs could include more context. Code impl looks fine tho.

Reply:
It looks like the pull request is stalling. The link to RFC 5929 has been added two days ago.

Reply:
Not stalling at all :-) ... just busy people working on lots of different things :-) I've started the CI on this, things are looking good so far.